.github/workflows/r-ci.yml

```diff
@@ -60,16 +60,28 @@ jobs:
       - name: Install BioConductor package SingleCellExperiment
         run: cd apis/r && tools/r-ci.sh install_bioc SingleCellExperiment
 
+      # The next two stanzas are necessary given propagation delay for binaries at
+      # https://r2u.stat.illinois.edu/ which may lack binaries (for short periods of usually a day)
+      # when sources have been updated. The default installation could switch to installation from
+      # sources --- but that would require installing all build dependencies. To see what the most
+      # recent binary is run e.g.  docker run --rm -ti rocker/r2u bash -c 'apt update -qq &&
+      # apt-cache show r-cran-tiledb' Using the r-universe builds (as below) is a suitable fallback
+      # as they update more frequently than CRAN.
```

Comment:
I am not really sure I agree with the wording.  Sadly I cannot quite put my finger on something better.  

But riffing for a moment we seem to have the three situations here:
- tiledb-r and tiledbsoma agree on versions, the world is good
- tiledb-r is
  - at release at CRAN (which is what we generally want)
  - at release-candidate in the GH repo and available via selected r-universe builds 
 
That last bit is "unusual but common" and I think the bit we have here today.  

And just how he have / had the existing ("sleeping") PR #1994 we have this now.   Part of me thinks we may want to discuss how to drive this better from one top-level (possibly outside the repo?) 'declarative' file. 

It's tricky.